CMakeLists.txt

```diff
@@ -78,3 +78,7 @@ add_test(NAME test_verilog COMMAND test_verilog WORKING_DIRECTORY ${CMAKE_SOURCE
 add_executable(test_inline_wires tests/gtest/test_inline_wires.cpp)
 target_link_libraries(test_inline_wires gtest_main coreir coreir-commonlib)
 add_test(NAME test_inline_wires COMMAND test_inline_wires WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tests/gtest)
+
```

Comment:
I must have missed this last time, but why are specific tests specified in the base CMakeLists.txt? Can we move this to a CMakeLists.txt in the test directory?

Reply:
Done